CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[v0.5.0\]
+
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
+- Integrate ov-telemetry (<https://github.com/openvinotoolkit/training_extensions/pull/1568>)
+
+### Changed
+
+- Update OpenVINO to 2022.3 release (<https://github.com/openvinotoolkit/training_extensions/pull/1393>)
```

Comment:
In this PR NNCF has been updated as well to the pre-2.4 version

Reply:
> In this PR NNCF has been updated as well to the pre-2.4 version

Updated.